test/unit_tests/apps/owning-a-home/js/explore-rates/dom-values-spec.js

```diff
@@ -0,0 +1,43 @@
+const BASE_JS_PATH = '../../../../../../cfgov/unprocessed/apps/owning-a-home/';
```

Comment:
Should this live in the base of `test/unit_tests/apps/owning-a-home` and be required in to each file that needs it?

Reply:
We could probably do something with https://nodejs.org/api/path.html, but that should apply to all tests so best in its own PR.